lib/src/assemble/manifest.rs

```diff
@@ -443,6 +444,13 @@ impl<'a> FakeDataAttributes<'a> {
                     "fake manifest should not have an installinator document"
                 );
             }
+            KnownArtifactKind::MeasurementCorpus => {
+                return make_filler_text(
```

Comment:
Do you plan to add some structure here in the future, similar to say the Hubris artifacts?